pkg/visibility/api/rest/pending_workloads_cq_test.go

```diff
@@ -149,8 +154,9 @@ func TestPendingWorkloadsInCQ(t *testing.T) {
 				},
 				{
 					ObjectMeta: v1.ObjectMeta{
-						Name:      "lqB-low-prio",
-						Namespace: nsName,
+						Name:            "lqB-low-prio",
+						Namespace:       nsName,
+						OwnerReferences: []metav1.OwnerReference{},
```

Comment:
you can use `cmpopts.EquateEmpty` to avoid this.

Or... just don't instantiate a list that is empty.